src/java/htsjdk/variant/variantcontext/CommonInfo.java

```diff
@@ -90,27 +91,43 @@ public void setName(String name) {
     //
     // ---------------------------------------------------------------------------------------------------------
 
+    /** @return a <b>modifiable</b> Set of filters. Can be null. All changes in this set will be reflected in the CommonInfo */
     public Set<String> getFiltersMaybeNull() {
         return filters;
     }
 
+    /** @return an unmodifiable Set of filters. Can be empty but never null */
     public Set<String> getFilters() {
         return filters == null ? NO_FILTERS : Collections.unmodifiableSet(filters);
     }
 
+    /** @return true if no filter has been defined  <code>(getFiltersMaybeNull()==null)</code> */
     public boolean filtersWereApplied() {
         return filters != null;
     }
 
+    /** @return true if any filter been defined  <code>(!filters.isEmpty())</code> */
     public boolean isFiltered() {
         return filters == null ? false : !filters.isEmpty();
     }
 
+    /**
+     * @param filter the filter ID 
+     * @return true if the filters contains 'filter' */
```

Comment:
Document that this method won't work for `PASS`, since it's not included in the set of filters